index.js

```diff
@@ -1,19 +1,19 @@
 var SecretStack = require('secret-stack')
-var create     = require('ssb-db/create')
-var ssbKeys    = require('ssb-keys')
-var path       = require('path')
-var osenv      = require('osenv')
-var mkdirp     = require('mkdirp')
-var rimraf     = require('rimraf')
-var mdm        = require('mdmanifest')
+var create = require('ssb-db/create')
+var ssbKeys = require('ssb-keys')
+var path = require('path')
+var osenv = require('osenv')
+var mkdirp = require('mkdirp')
+var rimraf = require('rimraf')
+var mdm = require('mdmanifest')
 var cmdAliases = require('./lib/cli-cmd-aliases')
-var valid      = require('./lib/validators')
-var apidocs    = require('./lib/apidocs.js')
-var pkg        = require('./package.json')
+var valid = require('./lib/validators')
+var apidocs = require('./lib/apidocs.js')
+var pkg = require('./package.json')
 
-function isString(s) { return 'string' === typeof s }
```

Comment:
yoda style is actually safer btw, because you can't accidentally do an assignment.
`missles = "fire"` vs `"fire" = missles`

Reply:
Standard doesn't allow `typeof s = 'string'`, just like it warns about things that will trip up ASI.